Polish login flow v2

kesselb opened the pull request:
**Master**
![master](https://user-images.githubusercontent.com/3902676/65817406-c12ace80-e206-11e9-8d5a-f57db6080ffa.png)

![master](https://user-images.githubusercontent.com/3902676/65817417-e3245100-e206-11e9-888b-ecb8ee7c04e4.png)

**This PR**
![this pr](https://user-images.githubusercontent.com/3902676/67598604-363de500-f76e-11e9-8ef7-45498861b1f2.png)

![this pr](https://user-images.githubusercontent.com/3902676/67598711-8a48c980-f76e-11e9-9e5d-09db3e02abbf.png)

kesselb committed:
Add icon-confirm-white to submit button

Signed-off-by: Daniel Kesselberg <mail@danielkesselberg.de>

kesselb committed:
Move "you can close this window" to newline

Signed-off-by: Daniel Kesselberg <mail@danielkesselberg.de>

jancborchardt reviewed (approved):
Looks better, thanks! :) There’s still that pesky issue of: Container on update screen is too wide #17556

kesselb commented on core/templates/loginflowv2/grant.php:
```diff
@@ -42,8 +42,7 @@
 			<input type="hidden" name="stateToken" value="<?php p($_['stateToken']) ?>" />	
 			<div id="submit-wrapper">
-				<input type="submit" id="submit" class="login primary" title="" value="<?php p($l->t('Grant access')); ?>" />
-				<div class="submit-icon icon-confirm-white"></div>
```
Hmm. https://github.com/nextcloud/server/search?q=submit-icon&unscoped_q=submit-icon `.submit-icon` is used by `grant.js` but this script is only loaded for loginflow v1.

kesselb commented:
Do you want this fancy loading animation back @jancborchardt ?

![Peek 2019-10-26 19-53](https://user-images.githubusercontent.com/3902676/67624016-300e3e00-f82c-11e9-8d22-203282985174.gif)

jancborchardt commented:
> Do you want this fancy loading animation back @jancborchardt ?
> 
> ![Peek 2019-10-26 19-53](https://user-images.githubusercontent.com/3902676/67624016-300e3e00-f82c-11e9-8d22-203282985174.gif)

Is that shown after tapping the button as feedback? If so then yes, feedback is always good. :)

kesselb committed:
The loading animation is back

Refactor grant.js to Vanilla JS
Remove id="submit" because of side effects: https://stackoverflow.com/questions/22982741/form-submit-jquery-does-not-work
Fix arrow for login flow v1

Signed-off-by: Daniel Kesselberg <mail@danielkesselberg.de>

kesselb commented:
OK. It was broken for login flow v1 either. Fixed both and rewrote it without jquery. Now we need some js power ;)

kesselb commented:
`grant.js` is not transpiled by babel :disappointed:  Checked `const, classList, forEach` with Can I use and it should work with: https://browserl.ist/?q=last+2+versions%2C+ie+%3E%3D+11%2C+firefox+ESR%2C+not+dead :see_no_evil:

kesselb commented:
> So we can merge this? Or should we transpile grant.js?

I think `grant.js` works for the supported browsers.
